demos/nvs-spi-flash/src/main.c

```diff
@@ -0,0 +1,268 @@
+/*
+ * NOTE: This demo is the official Zephyr demo adapted to suit the needs of efc
```

Comment:
We're not allowed to do this. `efc` is licensed under GPLv3, and the Zephyr sample is under Apache 2.0. You're not allowed to apply a more restrictive license to a piece of code AFAIK.

Reply:
As an alternative, why not suggest building the official sample with our board?

If that is not an option, we have to implement the sample from scratch, without looking at the Zephyr sample.